### ISSUE TITLE
Action: Create localized template for aria-label

### ISSUE THREAD

**Issue by driskull** [Esri/calcite-design-system#9071]
### Check existing issues

- [X] I have [checked for existing issues](https://github.com/Esri/calcite-design-system/issues) to avoid duplicates

### Actual Behavior

aria-label is using a template for its string value here

https://github.com/Esri/calcite-design-system/blob/15ca10a2bbedc568951246ff57537907e257bbc2/packages/calcite-components/src/components/action/action.tsx#L302

### Expected Behavior

This should use a proper localized template

### Reproduction Sample

https://codepen.io/driskull/pen/NWmyeXw?editors=1000

### Reproduction Steps

1. open codepen
2. Inspect internal element with aria-label `aria-label="test (Indicator present)"`

### Reproduction Version

1.7.1

### Relevant Info

https://github.com/Esri/calcite-design-system/pull/9051#discussion_r1551861189

### Priority impact

p4 - not time sensitive

### Calcite package

- [X] @esri/calcite-components
- [ ] @esri/calcite-components-angular
- [ ] @esri/calcite-components-react
- [ ] @esri/calcite-design-tokens
- [ ] @esri/eslint-plugin-calcite-components

### Esri team

N/A

**Comment by driskull** [comment]
@geospatialem can we move this issue until the string is available in all locales?

**Comment by geospatialem** [comment]
> @geospatialem can we move this issue until the string is available in all locales?

Sure, we're currently set for October, would this fit the timeline, or do we need to push to November, @driskull?

**Comment by driskull** [comment]
Just whenever the strings will be there.

**Comment by geospatialem** [comment]
> Just whenever the strings will be there.

checked in with Anvesh and we should be on target for October, but can move this out to November if the translations are delayed.

**Comment by driskull** [comment]
Sounds good!

**Comment by geospatialem** [comment]
Verified in `3.0.0-next.51` with `action`'s `label`, `text`, and fallback of an empty string when neither attr/prop is specified.
